www/docs/mp/index.php

```diff
@@ -489,6 +489,22 @@
         MySociety\TheyWorkForYou\Renderer::output('mp/divisions', $data);
 
         break;
+
+    case 'policy_svg':
+        $policy = get_http_var('policy');
```

Comment:
I was a bit confused here that it's using "policy" when it's actually a "set" or "topic"?